src/scipp/core/binning.py

```diff
@@ -202,11 +202,22 @@ def _upper_bound(x: Variable) -> Variable:
     return bound
 
 
+def _total_number_of_events_in_binned(x: DataArray):
+    return (x.bins.constituents['end'] - x.bins.constituents['begin']).sum().value
```

Comment:
Good, I'll use `x.bins.size().sum()` instead.

> Instead of counting events, can you just use the min and max and check those? 
I don't understand, do you mean `min` and `max` of the coordinate? 

With a single event we get a binning where every edge has the same value.
```python
import scipp as sc
import numpy as np
rng = np.random.default_rng(123)

lati = rng.random(10)
# modify a single entry to be in the 'lati' bin with index 1
lati[-1] = 1.5

da = sc.DataArray(
    data=sc.ones(sizes={"row": 10}, unit="counts"),
    coords={
        'longi': sc.array(dims=['row'], values=rng.random(10)),
        'lati': sc.array(dims=['row'], values=lati),
    }
)

empty_b = da.bin(lati=sc.array(dims=['lati'], values=[0.0, 1.2, 2.0]))['lati', 1]
empty_b.bin(longi=5)
# <scipp.DataArray>Dimensions: Sizes[longi:5, ] Coordinates: ... longi float64  [dimensionless]  (longi [bin-edge])  [0.518165, 0.518165, ..., 0.518165, 0.518165] Data: ...
```

Reply:
Yes, I mean use what is computed below anyway, and check if the range is valid?

Reply:
Aha okay, I'll try to do that instead.